### PR TITLE
Update readme intro data

### PR DESCRIPTION
**Pull request by jharenza**
#### What scientific question is your analysis addressing?
Adding information in README to better describe the OpenPBTA project, as well as data types and sequencing strategies available for analysis.

#### What GitHub issue does your pull request address?
[issue request](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/332) and [this comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/342/files#r358922674)

#### Which areas should receive a particularly close look?
Edited outside of doctoc, just wanted to be sure that was the right way to do it!

#### Is there anything that you want to discuss further?
Are we missing anything? I chose to focus on rna-seq for histologies because of the higher N (arbitrary) - could also make a table for WGS/WXS, but thought just listing them once is sufficient.

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
yes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,19 @@
 # OpenPBTA-analysis
 
-The Open Pediatric Brain Tumor Atlas (OpenPBTA) Project is an effort to describe the landscape of tumors in the [Children's Brain Tumor Tissue Consortium](https://cbttc.org/) and the PNOC003 DIPG clinical trial from the [Pediatric Pacific Neuro-oncology Consortium](http://www.pnoc.us/).
-This is an open analysis effort that is organized on GitHub.
-There is a [companion OpenPBTA-manuscript repository](https://github.com/AlexsLemonade/OpenPBTA-manuscript/) that is being used to author a collaborative manuscript describing the effort.
+Pediatric brain tumors are the most common solid tumors and the leading cause of cancer-related death in children. Small sample sizes due to the overall rarity of unique molecular subtypes and tainted grouped analyses resulting from misclassification have hindered our ability to understand and successfully treat these diseases.
```
Comment (cgreene) [inline README.md]:
```suggestion
Pediatric brain tumors are the most common solid tumors and the leading cause of cancer-related death in children.
Our ability to understand and successfully treat these diseases is hindered by small sample sizes due to the overall rarity of unique molecular subtypes and tainted grouped analyses resulting from misclassification. 
```

File: README.md [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
-This is an open analysis effort that is organized on GitHub.
-There is a [companion OpenPBTA-manuscript repository](https://github.com/AlexsLemonade/OpenPBTA-manuscript/) that is being used to author a collaborative manuscript describing the effort.
+Pediatric brain tumors are the most common solid tumors and the leading cause of cancer-related death in children. Small sample sizes due to the overall rarity of unique molecular subtypes and tainted grouped analyses resulting from misclassification have hindered our ability to understand and successfully treat these diseases. 
+In September of 2018, the [Children's Brain Tumor Tissue Consortium](https://cbttc.org/) released the [Pediatric Brain Tumor Atlas (PBTA)](https://cbttc.org/pediatric-brain-tumor-atlas/), a genomic dataset (WGS, RNA-Seq, proteomics, and clinical data) for nearly 1,000 tumors, available from the [Gabriella Miller Kids First Portal](https://kidsfirstdrc.org/).
```
Comment (cgreene) [inline README.md]:
```suggestion
In September of 2018, the [Children's Brain Tumor Tissue Consortium](https://cbttc.org/) released the [Pediatric Brain Tumor Atlas (PBTA)](https://cbttc.org/pediatric-brain-tumor-atlas/), a genomic dataset (whole genome sequencing, whole exome sequencing, RNA sequencing, proteomic, and clinical data) for nearly 1,000 tumors, available from the [Gabriella Miller Kids First Portal](https://kidsfirstdrc.org/). 
```

File: README.md [hunk 3]
```diff
@@ -41,14 +46,72 @@ The project maintainers include scientists from [Alex's Lemonade Stand Foundatio
...
+## Data Description
+
+The OpenPBTA dataset includes gene expression, fusion, as ell as somatic mutation, copy number, structural and variant results in combined tsv or matrix format.
+
+Below is a summary of biospecimens by sequencing strategy:
+
+
+| Experimental Strategy | Normal | Tumor |
+|-----------------------|--------|-------|
+| Panel | 1 | 1 |
```
Comment (cgreene) [inline README.md]: `Panel` is unclear here.

File: README.md [hunk 4]
```diff
@@ -41,14 +46,72 @@ The project maintainers include scientists from [Alex's Lemonade Stand Foundatio
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Data Description
+
+The OpenPBTA dataset includes gene expression, fusion, as ell as somatic mutation, copy number, structural and variant results in combined tsv or matrix format.
```
Comment (cgreene) [inline README.md]:
```suggestion
The OpenPBTA dataset includes gene expression, fusion, as well as somatic mutation, copy number, structural and variant results in combined tsv or matrix format.
```

File: README.md [hunk 5]
```diff
@@ -1,14 +1,19 @@
...
+In September of 2018, the [Children's Brain Tumor Tissue Consortium](https://cbttc.org/) released the [Pediatric Brain Tumor Atlas (PBTA)](https://cbttc.org/pediatric-brain-tumor-atlas/), a genomic dataset (WGS, RNA-Seq, proteomics, and clinical data) for nearly 1,000 tumors, available from the [Gabriella Miller Kids First Portal](https://kidsfirstdrc.org/). 
+
+The Open Pediatric Brain Tumor Atlas (OpenPBTA) Project is a global open science initiative to comprehensively define the molecular landscape of tumors of 944 patients from the CBTTC and the PNOC003 DIPG clinical trial from the [Pediatric Pacific Neuro-oncology Consortium](http://www.pnoc.us/) through real-time, collaborative analyses and [collaborative manuscript writing](https://github.com/AlexsLemonade/OpenPBTA-manuscript/) on GitHub. 
+
+The OpenPBTA uses a pull request system to accept contributions from community participants. Continuous integration software is applied to test datasets to confirm reproducibility of analyses within the project’s Docker container. Collaborative manuscript writing employs the Manubot software to provide an up-to-date public version of the manuscript.
```
Comment (cgreene) [inline README.md]:
```suggestion
The OpenPBTA operates on a pull request model to accept contributions from community participants.
The maintainers have setup continuous integration software to confirm the reproducibility of analyses within the project’s Docker container.
The collaborative manuscript is authored using [Manubot](https://manubot.org) software to provide an up-to-date public version of the manuscript. 
```

File: README.md [hunk 6]
```diff
@@ -41,14 +46,72 @@ The project maintainers include scientists from [Alex's Lemonade Stand Foundatio
...
+| Panel | 1 | 1 |
+| RNA-Seq | 0 | 1072 |
+| WGS | 801 | 940 |
+| WXS | 31 | 31 |
+
+
+All sequencing data were performed on nucleic acids extracted from fresh-frozen tissues using paired-end strategies.
+Additional details can be found in the [manuscript methods](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#data-generation).
+
+Below is a detailed table of [broad histologies](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#who-classification-of-disease-types) and numbers for the 1072 RNA-Seq biospecimens:
```
Comment (cgreene) [inline README.md]:
```suggestion
Below is a detailed table of [broad histologies](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#who-classification-of-disease-types) for the 1072 RNA-Seq biospecimens:
```

File: README.md [hunk 7]
```diff
@@ -41,14 +46,72 @@ The project maintainers include scientists from [Alex's Lemonade Stand Foundatio
...
+| Experimental Strategy | Normal | Tumor |
+|-----------------------|--------|-------|
+| Panel | 1 | 1 |
+| RNA-Seq | 0 | 1072 |
+| WGS | 801 | 940 |
+| WXS | 31 | 31 |
+
+
+All sequencing data were performed on nucleic acids extracted from fresh-frozen tissues using paired-end strategies.
+Additional details can be found in the [manuscript methods](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#data-generation).
```
Comment (cgreene) [inline README.md]:
```suggestion
The [manuscript methods section](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#data-generation) has additional details.
```

File: README.md [hunk 8]
```diff
@@ -41,14 +46,72 @@ The project maintainers include scientists from [Alex's Lemonade Stand Foundatio
...
+
+| Experimental Strategy | Normal | Tumor |
+|-----------------------|--------|-------|
+| Panel | 1 | 1 |
+| RNA-Seq | 0 | 1072 |
+| WGS | 801 | 940 |
+| WXS | 31 | 31 |
+
+
+All sequencing data were performed on nucleic acids extracted from fresh-frozen tissues using paired-end strategies.
```
Comment (cgreene) [inline README.md]:
```suggestion
All sequencing was performed on nucleic acids extracted from fresh-frozen tissues using paired-end strategies.
```

File: README.md (hunk 3 above)
Comment (jharenza) [reply README.md]: updated to `Targeted DNA Panel`

**Pull request merged**